docs/upgrade/0.3.5-to-0.3.6.rst

```diff
@@ -0,0 +1,100 @@
+Upgrade from 0.3.5 to 0.3.6
+============================
+
+SecureDrop 0.3.6 was a maintenance release to update the expiration
+date on the GPG signing key used for the Freedom of the Press
+apt repository. The signing key expired on October 26, 2015, preventing
+automatically updates from Freedom of the Press until the issue is
+manually resolved.
+
+
+.. note:: We're considering creating a separate ``securedrop-keyring`` 
+      Debian package to package to manage the signing keys into the future. 
+      If you have strong feelings about how the signing key is managed,
+      `open an issue <https://github.com/freedomofpress/securedrop/issues/>`_.
+
+
+The upgrade steps in this document will upgrade a 0.3.5 SecureDrop to
+0.3.6. If you have not yet upgraded to 0.3.5, :doc:`do that first <0.3.x-to-0.3.5>`.
+
+Important Changes
+-----------------
+
+Update Expired GPG Signing Key (Admin Workstation)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Admin Workstation is used to verify the signed git tag on new
+SecureDrop releases. Prior to running the 0.3.6 upgrade, the Admin
+Workstation should update the signing key.
+
+Update Expired GPG Signing Key (SecureDrop Servers)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Application Server and Monitor Server will need to have their apt
+keyrings updated, which can be accomplished by running the Ansible playbooks.
+
+
+Prerequisites
+-------------
+
+-  An Admin networked Tails workstation with persistence enabled and an
+   Admin password set during boot.
+   
+-  A running SecureDrop instance of 0.3.5.
+
+.. note:: If your SecureDrop instance is not already running 0.3.5,
+          :doc:`run that upgrade first <0.3.x-to-0.3.5>`. Failure to do 
+          so will result in a merge conflict for your site-specific
+          settings, which can be tedious to resolve if you're not
+          experienced with git.
+
+Upgrade Procedure
+-----------------
+
+The upgrade procedure can be performed entirely from the Admin
+Workstation. Start by booting the Admin Tails USB on the Admin 
+Workstation. Make sure you set an *Adminstrator Password* in the *Tails Greeter*.
+
+#. Open a terminal (it is an icon on the top of the screen that looks
+   like a little black TV).
+
+#. Change into the SecureDrop repo directory ::
+
+     cd /home/amnesia/Persistent/securedrop
+
+#. Stash your local changes to the inventory and prod-specific.yml ::
+
+     git stash save "site specific configs"
+
+#. Update the Freedom of the Press signing key
+   (these commands are from the :doc:`../set_up_admin_tails` guide) ::
+
+    gpg --keyserver pool.sks-keyservers.net --recv-key B89A29DB2128160B8E4B1B4CBADDE0C7FC9F6818
+    gpg --fingerprint B89A29DB2128160B8E4B1B4CBADDE0C7FC9F6818
```

Comment:
We should add a note, here or as a link from here to a dedicate document, about verifying this key. It is available on numerous HTTPS sites, and it is also signed by all of the developer's personal keys, so it should be possible to verify it via the GPG WoT as well.